src/lib/renderComponent.tsx

```diff
@@ -58,11 +60,47 @@ const getAccessibility = <P extends {}>(props, state) => {
   })
 }
 
+/**
```

Comment:
I think this function is now long enough to be separated to a separate file (similarly to how renderComponent is in its own ts file

Reply:
Agree, will move to separate file